2019-12-24 01:06:55 [scrapy] DEBUG: Retrying <GET https://indeed.fr/robots.txt> (failed 1 times): [<twisted.python.failure.Failure twisted.internet.error.ConnectionLost: Connection to the other side was lost in a non-clean fashion: Connection lost.>]
2019-12-24 01:07:07 [scrapy] DEBUG: Retrying <GET https://indeed.fr/robots.txt> (failed 2 times): [<twisted.python.failure.Failure twisted.internet.error.ConnectionLost: Connection to the other side was lost in a non-clean fashion: Connection lost.>]
2019-12-24 01:07:19 [scrapy] DEBUG: Gave up retrying <GET https://indeed.fr/robots.txt> (failed 3 times): [<twisted.python.failure.Failure twisted.internet.error.ConnectionLost: Connection to the other side was lost in a non-clean fashion: Connection lost.>]
2019-12-24 01:07:19 [scrapy] ERROR: Error downloading <GET https://indeed.fr/robots.txt>: [<twisted.python.failure.Failure twisted.internet.error.ConnectionLost: Connection to the other side was lost in a non-clean fashion: Connection lost.>]
Traceback (most recent call last):
  File "/usr/local/lib/python3.6/dist-packages/scrapy/core/downloader/middleware.py", line 44, in process_request
    defer.returnValue((yield download_func(request=request, spider=spider)))
twisted.web._newclient.ResponseNeverReceived: [<twisted.python.failure.Failure twisted.internet.error.ConnectionLost: Connection to the other side was lost in a non-clean fashion: Connection lost.>]
2019-12-24 01:07:32 [scrapy] DEBUG: Retrying <GET https://indeed.fr/> (failed 1 times): [<twisted.python.failure.Failure twisted.internet.error.ConnectionLost: Connection to the other side was lost in a non-clean fashion: Connection lost.>]
2019-12-24 01:07:44 [scrapy] DEBUG: Retrying <GET https://indeed.fr/> (failed 2 times): [<twisted.python.failure.Failure twisted.internet.error.ConnectionLost: Connection to the other side was lost in a non-clean fashion: Connection lost.>]
2019-12-24 01:07:57 [scrapy] DEBUG: Gave up retrying <GET https://indeed.fr/> (failed 3 times): [<twisted.python.failure.Failure twisted.internet.error.ConnectionLost: Connection to the other side was lost in a non-clean fashion: Connection lost.>]
2019-12-24 01:07:57 [scrapy] ERROR: Error downloading <GET https://indeed.fr/>
Traceback (most recent call last):
  File "/usr/local/lib/python3.6/dist-packages/scrapy/core/downloader/middleware.py", line 44, in process_request
    defer.returnValue((yield download_func(request=request, spider=spider)))
twisted.web._newclient.ResponseNeverReceived: [<twisted.python.failure.Failure twisted.internet.error.ConnectionLost: Connection to the other side was lost in a non-clean fashion: Connection lost.>]
2019-12-24 01:07:57 [scrapy] INFO: Closing spider (finished)
2019-12-24 01:07:57 [scrapy] INFO: Dumping Scrapy stats:
{'downloader/exception_count': 6,
 'downloader/exception_type_count/twisted.web._newclient.ResponseNeverReceived': 6,
 'downloader/request_bytes': 1278,
 'downloader/request_count': 6,
 'downloader/request_method_count/GET': 6,
 'elapsed_time_seconds': 76.049262,
 'finish_reason': 'finished',
 'finish_time': datetime.datetime(2019, 12, 24, 0, 7, 57, 374723),
 'log_count/DEBUG': 6,
 'log_count/ERROR': 2,
 'log_count/INFO': 7,
 'memusage/max': 54857728,
 'memusage/startup': 53899264,
 'retry/count': 4,
 'retry/max_reached': 2,
 'retry/reason_count/twisted.web._newclient.ResponseNeverReceived': 4,
 "robotstxt/exception_count/<class 'twisted.web._newclient.ResponseNeverReceived'>": 1,
 'robotstxt/request_count': 1,
 'scheduler/dequeued': 3,
 'scheduler/dequeued/memory': 3,
 'scheduler/enqueued': 3,
 'scheduler/enqueued/memory': 3,
 'start_time': datetime.datetime(2019, 12, 24, 0, 6, 41, 325461)}
2019-12-24 01:07:57 [scrapy] INFO: Spider closed (finished)
